
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="http://docs.franklin.hpc.ucdavis.edu/software/list/">
      
      
        <link rel="prev" href="../">
      
      
        <link rel="next" href="../modules/">
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.0, mkdocs-material-9.5.27">
    
    
      
        <title>Software List - Franklin Cluster Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.6543a935.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="ucdavis" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#spack-managed-modules" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Franklin Cluster Documentation" class="md-header__button md-logo" aria-label="Franklin Cluster Documentation" data-md-component="logo">
      
  <img src="../../assets/HPCCF-logo-dark.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Franklin Cluster Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Software List
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="ucdavis" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12c0-2.42-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/ucdavis/HPCCF-franklin-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Franklin Cluster Documentation" class="md-nav__button md-logo" aria-label="Franklin Cluster Documentation" data-md-component="logo">
      
  <img src="../../assets/HPCCF-logo-dark.png" alt="logo">

    </a>
    Franklin Cluster Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/ucdavis/HPCCF-franklin-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    About
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
      
        
      
        
      
        
      
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Users
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Users
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
    
    
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_2_1" >
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../general/" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    General
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2_1" id="__nav_2_1_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_1">
            <span class="md-nav__icon md-icon"></span>
            General
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../general/access/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Accessing the Cluster
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../general/environment.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    User Environment
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../general/resources.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Available Resources
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../general/storage.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data Storage
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
    
    
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_2_2" >
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../scheduler/" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    Scheduler
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2_2" id="__nav_2_2_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_2">
            <span class="md-nav__icon md-icon"></span>
            Scheduler
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../scheduler/jobs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Slurm Commands
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../scheduler/queues/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Queueing
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../scheduler/resources/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Resources
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../scheduler/jobscripts/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Job Scripts
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
    
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" checked>
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    Software
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Software
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Software List
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Software List
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#r" class="md-nav__link">
    <span class="md-ellipsis">
      R
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abyss" class="md-nav__link">
    <span class="md-ellipsis">
      abyss
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amdfftw" class="md-nav__link">
    <span class="md-ellipsis">
      amdfftw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ant" class="md-nav__link">
    <span class="md-ellipsis">
      ant
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aragorn" class="md-nav__link">
    <span class="md-ellipsis">
      aragorn
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedtools2" class="md-nav__link">
    <span class="md-ellipsis">
      bedtools2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blast-plus" class="md-nav__link">
    <span class="md-ellipsis">
      blast-plus
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blast2go" class="md-nav__link">
    <span class="md-ellipsis">
      blast2go
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blat" class="md-nav__link">
    <span class="md-ellipsis">
      blat
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie" class="md-nav__link">
    <span class="md-ellipsis">
      bowtie
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie2" class="md-nav__link">
    <span class="md-ellipsis">
      bowtie2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwa" class="md-nav__link">
    <span class="md-ellipsis">
      bwa
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwtool" class="md-nav__link">
    <span class="md-ellipsis">
      bwtool
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#canu" class="md-nav__link">
    <span class="md-ellipsis">
      canu
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cap3" class="md-nav__link">
    <span class="md-ellipsis">
      cap3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustal-omega" class="md-nav__link">
    <span class="md-ellipsis">
      clustal-omega
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustalw" class="md-nav__link">
    <span class="md-ellipsis">
      clustalw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#corset" class="md-nav__link">
    <span class="md-ellipsis">
      corset
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctffind" class="md-nav__link">
    <span class="md-ellipsis">
      ctffind
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cuda" class="md-nav__link">
    <span class="md-ellipsis">
      cuda
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cufflinks" class="md-nav__link">
    <span class="md-ellipsis">
      cufflinks
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ea-utils" class="md-nav__link">
    <span class="md-ellipsis">
      ea-utils
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emboss" class="md-nav__link">
    <span class="md-ellipsis">
      emboss
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerate" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerate-gff3" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate-gff3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastqc" class="md-nav__link">
    <span class="md-ellipsis">
      fastqc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fftw" class="md-nav__link">
    <span class="md-ellipsis">
      fftw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freebayes" class="md-nav__link">
    <span class="md-ellipsis">
      freebayes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gatk" class="md-nav__link">
    <span class="md-ellipsis">
      gatk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gcc" class="md-nav__link">
    <span class="md-ellipsis">
      gcc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gctf" class="md-nav__link">
    <span class="md-ellipsis">
      gctf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genrich" class="md-nav__link">
    <span class="md-ellipsis">
      genrich
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ghostscript" class="md-nav__link">
    <span class="md-ellipsis">
      ghostscript
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glimmer" class="md-nav__link">
    <span class="md-ellipsis">
      glimmer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf5" class="md-nav__link">
    <span class="md-ellipsis">
      hdf5
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hisat2" class="md-nav__link">
    <span class="md-ellipsis">
      hisat2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmer" class="md-nav__link">
    <span class="md-ellipsis">
      hmmer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#homer" class="md-nav__link">
    <span class="md-ellipsis">
      homer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hwloc" class="md-nav__link">
    <span class="md-ellipsis">
      hwloc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#igv" class="md-nav__link">
    <span class="md-ellipsis">
      igv
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#infernal" class="md-nav__link">
    <span class="md-ellipsis">
      infernal
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-oneapi-compilers" class="md-nav__link">
    <span class="md-ellipsis">
      intel-oneapi-compilers
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-oneapi-mkl" class="md-nav__link">
    <span class="md-ellipsis">
      intel-oneapi-mkl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#interproscan" class="md-nav__link">
    <span class="md-ellipsis">
      interproscan
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iq-tree" class="md-nav__link">
    <span class="md-ellipsis">
      iq-tree
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iqtree2" class="md-nav__link">
    <span class="md-ellipsis">
      iqtree2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jdk" class="md-nav__link">
    <span class="md-ellipsis">
      jdk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jellyfish" class="md-nav__link">
    <span class="md-ellipsis">
      jellyfish
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kalign" class="md-nav__link">
    <span class="md-ellipsis">
      kalign
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kallisto" class="md-nav__link">
    <span class="md-ellipsis">
      kallisto
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kmergenie" class="md-nav__link">
    <span class="md-ellipsis">
      kmergenie
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken" class="md-nav__link">
    <span class="md-ellipsis">
      kraken
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken2" class="md-nav__link">
    <span class="md-ellipsis">
      kraken2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#krakenuniq" class="md-nav__link">
    <span class="md-ellipsis">
      krakenuniq
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#last" class="md-nav__link">
    <span class="md-ellipsis">
      last
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libevent" class="md-nav__link">
    <span class="md-ellipsis">
      libevent
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mash" class="md-nav__link">
    <span class="md-ellipsis">
      mash
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#masurca" class="md-nav__link">
    <span class="md-ellipsis">
      masurca
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcl" class="md-nav__link">
    <span class="md-ellipsis">
      mcl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megahit" class="md-nav__link">
    <span class="md-ellipsis">
      megahit
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meme" class="md-nav__link">
    <span class="md-ellipsis">
      meme
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaeuk" class="md-nav__link">
    <span class="md-ellipsis">
      metaeuk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minced" class="md-nav__link">
    <span class="md-ellipsis">
      minced
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniasm" class="md-nav__link">
    <span class="md-ellipsis">
      miniasm
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimap2" class="md-nav__link">
    <span class="md-ellipsis">
      minimap2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mirdeep2" class="md-nav__link">
    <span class="md-ellipsis">
      mirdeep2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mmseqs2" class="md-nav__link">
    <span class="md-ellipsis">
      mmseqs2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mothur" class="md-nav__link">
    <span class="md-ellipsis">
      mothur
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#motioncor2" class="md-nav__link">
    <span class="md-ellipsis">
      motioncor2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummer" class="md-nav__link">
    <span class="md-ellipsis">
      mummer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummer4" class="md-nav__link">
    <span class="md-ellipsis">
      mummer4
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muscle" class="md-nav__link">
    <span class="md-ellipsis">
      muscle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-rmblastn" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-rmblastn
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-toolkit" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-toolkit
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-vdb" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-vdb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextflow" class="md-nav__link">
    <span class="md-ellipsis">
      nextflow
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openjdk" class="md-nav__link">
    <span class="md-ellipsis">
      openjdk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openldap" class="md-nav__link">
    <span class="md-ellipsis">
      openldap
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openmpi" class="md-nav__link">
    <span class="md-ellipsis">
      openmpi
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofinder" class="md-nav__link">
    <span class="md-ellipsis">
      orthofinder
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthomcl" class="md-nav__link">
    <span class="md-ellipsis">
      orthomcl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallel" class="md-nav__link">
    <span class="md-ellipsis">
      parallel
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#patchelf" class="md-nav__link">
    <span class="md-ellipsis">
      patchelf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phylip" class="md-nav__link">
    <span class="md-ellipsis">
      phylip
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#picard" class="md-nav__link">
    <span class="md-ellipsis">
      picard
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pilon" class="md-nav__link">
    <span class="md-ellipsis">
      pilon
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    <span class="md-ellipsis">
      plink
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pmix" class="md-nav__link">
    <span class="md-ellipsis">
      pmix
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prokka" class="md-nav__link">
    <span class="md-ellipsis">
      prokka
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r_1" class="md-nav__link">
    <span class="md-ellipsis">
      r
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxml-ng" class="md-nav__link">
    <span class="md-ellipsis">
      raxml-ng
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ray" class="md-nav__link">
    <span class="md-ellipsis">
      ray
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rclone" class="md-nav__link">
    <span class="md-ellipsis">
      rclone
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#recon" class="md-nav__link">
    <span class="md-ellipsis">
      recon
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion" class="md-nav__link">
    <span class="md-ellipsis">
      relion
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion-bbr" class="md-nav__link">
    <span class="md-ellipsis">
      relion-bbr
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion-helper" class="md-nav__link">
    <span class="md-ellipsis">
      relion-helper
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmasker" class="md-nav__link">
    <span class="md-ellipsis">
      repeatmasker
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmodeler" class="md-nav__link">
    <span class="md-ellipsis">
      repeatmodeler
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatscout" class="md-nav__link">
    <span class="md-ellipsis">
      repeatscout
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnaquast" class="md-nav__link">
    <span class="md-ellipsis">
      rnaquast
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsem" class="md-nav__link">
    <span class="md-ellipsis">
      rsem
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rstudio-server" class="md-nav__link">
    <span class="md-ellipsis">
      rstudio-server
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sabre" class="md-nav__link">
    <span class="md-ellipsis">
      sabre
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#satsuma2" class="md-nav__link">
    <span class="md-ellipsis">
      satsuma2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      scallop
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqprep" class="md-nav__link">
    <span class="md-ellipsis">
      seqprep
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqtk" class="md-nav__link">
    <span class="md-ellipsis">
      seqtk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sickle" class="md-nav__link">
    <span class="md-ellipsis">
      sickle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slurm" class="md-nav__link">
    <span class="md-ellipsis">
      slurm
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smartdenovo" class="md-nav__link">
    <span class="md-ellipsis">
      smartdenovo
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sortmerna" class="md-nav__link">
    <span class="md-ellipsis">
      sortmerna
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimmomatic" class="md-nav__link">
    <span class="md-ellipsis">
      trimmomatic
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ucx" class="md-nav__link">
    <span class="md-ellipsis">
      ucx
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../modules/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Module System
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../conda/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Python and Conda
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../rlang/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    R and RStudio
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../cryoem/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    cryo-EM
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../developing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Developing Software
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Admins
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Admins
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../admin_storage.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Storage
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../admin_networking.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Networking
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../admin_scheduler.md" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Scheduler
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#r" class="md-nav__link">
    <span class="md-ellipsis">
      R
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abyss" class="md-nav__link">
    <span class="md-ellipsis">
      abyss
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amdfftw" class="md-nav__link">
    <span class="md-ellipsis">
      amdfftw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ant" class="md-nav__link">
    <span class="md-ellipsis">
      ant
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aragorn" class="md-nav__link">
    <span class="md-ellipsis">
      aragorn
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedtools2" class="md-nav__link">
    <span class="md-ellipsis">
      bedtools2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blast-plus" class="md-nav__link">
    <span class="md-ellipsis">
      blast-plus
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blast2go" class="md-nav__link">
    <span class="md-ellipsis">
      blast2go
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blat" class="md-nav__link">
    <span class="md-ellipsis">
      blat
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie" class="md-nav__link">
    <span class="md-ellipsis">
      bowtie
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie2" class="md-nav__link">
    <span class="md-ellipsis">
      bowtie2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwa" class="md-nav__link">
    <span class="md-ellipsis">
      bwa
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwtool" class="md-nav__link">
    <span class="md-ellipsis">
      bwtool
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#canu" class="md-nav__link">
    <span class="md-ellipsis">
      canu
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cap3" class="md-nav__link">
    <span class="md-ellipsis">
      cap3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustal-omega" class="md-nav__link">
    <span class="md-ellipsis">
      clustal-omega
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustalw" class="md-nav__link">
    <span class="md-ellipsis">
      clustalw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#corset" class="md-nav__link">
    <span class="md-ellipsis">
      corset
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctffind" class="md-nav__link">
    <span class="md-ellipsis">
      ctffind
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cuda" class="md-nav__link">
    <span class="md-ellipsis">
      cuda
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cufflinks" class="md-nav__link">
    <span class="md-ellipsis">
      cufflinks
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ea-utils" class="md-nav__link">
    <span class="md-ellipsis">
      ea-utils
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emboss" class="md-nav__link">
    <span class="md-ellipsis">
      emboss
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerate" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerate-gff3" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate-gff3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastqc" class="md-nav__link">
    <span class="md-ellipsis">
      fastqc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fftw" class="md-nav__link">
    <span class="md-ellipsis">
      fftw
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freebayes" class="md-nav__link">
    <span class="md-ellipsis">
      freebayes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gatk" class="md-nav__link">
    <span class="md-ellipsis">
      gatk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gcc" class="md-nav__link">
    <span class="md-ellipsis">
      gcc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gctf" class="md-nav__link">
    <span class="md-ellipsis">
      gctf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genrich" class="md-nav__link">
    <span class="md-ellipsis">
      genrich
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ghostscript" class="md-nav__link">
    <span class="md-ellipsis">
      ghostscript
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glimmer" class="md-nav__link">
    <span class="md-ellipsis">
      glimmer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf5" class="md-nav__link">
    <span class="md-ellipsis">
      hdf5
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hisat2" class="md-nav__link">
    <span class="md-ellipsis">
      hisat2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmer" class="md-nav__link">
    <span class="md-ellipsis">
      hmmer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#homer" class="md-nav__link">
    <span class="md-ellipsis">
      homer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hwloc" class="md-nav__link">
    <span class="md-ellipsis">
      hwloc
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#igv" class="md-nav__link">
    <span class="md-ellipsis">
      igv
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#infernal" class="md-nav__link">
    <span class="md-ellipsis">
      infernal
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-oneapi-compilers" class="md-nav__link">
    <span class="md-ellipsis">
      intel-oneapi-compilers
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-oneapi-mkl" class="md-nav__link">
    <span class="md-ellipsis">
      intel-oneapi-mkl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#interproscan" class="md-nav__link">
    <span class="md-ellipsis">
      interproscan
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iq-tree" class="md-nav__link">
    <span class="md-ellipsis">
      iq-tree
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iqtree2" class="md-nav__link">
    <span class="md-ellipsis">
      iqtree2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jdk" class="md-nav__link">
    <span class="md-ellipsis">
      jdk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jellyfish" class="md-nav__link">
    <span class="md-ellipsis">
      jellyfish
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kalign" class="md-nav__link">
    <span class="md-ellipsis">
      kalign
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kallisto" class="md-nav__link">
    <span class="md-ellipsis">
      kallisto
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kmergenie" class="md-nav__link">
    <span class="md-ellipsis">
      kmergenie
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken" class="md-nav__link">
    <span class="md-ellipsis">
      kraken
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken2" class="md-nav__link">
    <span class="md-ellipsis">
      kraken2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#krakenuniq" class="md-nav__link">
    <span class="md-ellipsis">
      krakenuniq
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#last" class="md-nav__link">
    <span class="md-ellipsis">
      last
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libevent" class="md-nav__link">
    <span class="md-ellipsis">
      libevent
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mash" class="md-nav__link">
    <span class="md-ellipsis">
      mash
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#masurca" class="md-nav__link">
    <span class="md-ellipsis">
      masurca
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcl" class="md-nav__link">
    <span class="md-ellipsis">
      mcl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megahit" class="md-nav__link">
    <span class="md-ellipsis">
      megahit
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meme" class="md-nav__link">
    <span class="md-ellipsis">
      meme
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaeuk" class="md-nav__link">
    <span class="md-ellipsis">
      metaeuk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minced" class="md-nav__link">
    <span class="md-ellipsis">
      minced
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniasm" class="md-nav__link">
    <span class="md-ellipsis">
      miniasm
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimap2" class="md-nav__link">
    <span class="md-ellipsis">
      minimap2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mirdeep2" class="md-nav__link">
    <span class="md-ellipsis">
      mirdeep2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mmseqs2" class="md-nav__link">
    <span class="md-ellipsis">
      mmseqs2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mothur" class="md-nav__link">
    <span class="md-ellipsis">
      mothur
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#motioncor2" class="md-nav__link">
    <span class="md-ellipsis">
      motioncor2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummer" class="md-nav__link">
    <span class="md-ellipsis">
      mummer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummer4" class="md-nav__link">
    <span class="md-ellipsis">
      mummer4
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muscle" class="md-nav__link">
    <span class="md-ellipsis">
      muscle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-rmblastn" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-rmblastn
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-toolkit" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-toolkit
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-vdb" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-vdb
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextflow" class="md-nav__link">
    <span class="md-ellipsis">
      nextflow
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openjdk" class="md-nav__link">
    <span class="md-ellipsis">
      openjdk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openldap" class="md-nav__link">
    <span class="md-ellipsis">
      openldap
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openmpi" class="md-nav__link">
    <span class="md-ellipsis">
      openmpi
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofinder" class="md-nav__link">
    <span class="md-ellipsis">
      orthofinder
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthomcl" class="md-nav__link">
    <span class="md-ellipsis">
      orthomcl
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallel" class="md-nav__link">
    <span class="md-ellipsis">
      parallel
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#patchelf" class="md-nav__link">
    <span class="md-ellipsis">
      patchelf
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phylip" class="md-nav__link">
    <span class="md-ellipsis">
      phylip
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#picard" class="md-nav__link">
    <span class="md-ellipsis">
      picard
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pilon" class="md-nav__link">
    <span class="md-ellipsis">
      pilon
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    <span class="md-ellipsis">
      plink
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pmix" class="md-nav__link">
    <span class="md-ellipsis">
      pmix
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prokka" class="md-nav__link">
    <span class="md-ellipsis">
      prokka
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r_1" class="md-nav__link">
    <span class="md-ellipsis">
      r
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxml-ng" class="md-nav__link">
    <span class="md-ellipsis">
      raxml-ng
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ray" class="md-nav__link">
    <span class="md-ellipsis">
      ray
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rclone" class="md-nav__link">
    <span class="md-ellipsis">
      rclone
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#recon" class="md-nav__link">
    <span class="md-ellipsis">
      recon
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion" class="md-nav__link">
    <span class="md-ellipsis">
      relion
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion-bbr" class="md-nav__link">
    <span class="md-ellipsis">
      relion-bbr
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion-helper" class="md-nav__link">
    <span class="md-ellipsis">
      relion-helper
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmasker" class="md-nav__link">
    <span class="md-ellipsis">
      repeatmasker
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmodeler" class="md-nav__link">
    <span class="md-ellipsis">
      repeatmodeler
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatscout" class="md-nav__link">
    <span class="md-ellipsis">
      repeatscout
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnaquast" class="md-nav__link">
    <span class="md-ellipsis">
      rnaquast
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsem" class="md-nav__link">
    <span class="md-ellipsis">
      rsem
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rstudio-server" class="md-nav__link">
    <span class="md-ellipsis">
      rstudio-server
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sabre" class="md-nav__link">
    <span class="md-ellipsis">
      sabre
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#satsuma2" class="md-nav__link">
    <span class="md-ellipsis">
      satsuma2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scallop" class="md-nav__link">
    <span class="md-ellipsis">
      scallop
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqprep" class="md-nav__link">
    <span class="md-ellipsis">
      seqprep
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqtk" class="md-nav__link">
    <span class="md-ellipsis">
      seqtk
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sickle" class="md-nav__link">
    <span class="md-ellipsis">
      sickle
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slurm" class="md-nav__link">
    <span class="md-ellipsis">
      slurm
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smartdenovo" class="md-nav__link">
    <span class="md-ellipsis">
      smartdenovo
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sortmerna" class="md-nav__link">
    <span class="md-ellipsis">
      sortmerna
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimmomatic" class="md-nav__link">
    <span class="md-ellipsis">
      trimmomatic
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ucx" class="md-nav__link">
    <span class="md-ellipsis">
      ucx
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="spack-managed-modules">Spack-managed Modules<a class="headerlink" href="#spack-managed-modules" title="Permanent link">&para;</a></h1>
<p>These modules are built and managed by our Spack deployment.
Most were compiled for generic architecture, meaning they can run on any node,
but some are Intel or AMD specific, and some require GPU support.</p>
<h2 id="r">R<a class="headerlink" href="#r" title="Permanent link">&para;</a></h2>
<p>R is 'GNU S', a freely available language and environment for
statistical computing and graphics which provides a wide variety of
statistical and graphical techniques: linear and nonlinear modelling,
statistical tests, time series analysis, classification, clustering,
etc. Please consult the R project homepage for further information.</p>
<p><strong>Versions</strong>: 4.1.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>R/4.1.1</code></p>
<h2 id="abyss">abyss<a class="headerlink" href="#abyss" title="Permanent link">&para;</a></h2>
<p>ABySS is a de novo, parallel, paired-end sequence assembler that is
designed for short reads. The single-processor version is useful for
assembling genomes up to 100 Mbases in size.</p>
<p><strong>Versions</strong>: 2.3.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>abyss/2.3.1</code></p>
<h2 id="amdfftw">amdfftw<a class="headerlink" href="#amdfftw" title="Permanent link">&para;</a></h2>
<p>FFTW (AMD Optimized version) is a comprehensive collection of fast C
routines for computing the Discrete Fourier Transform (DFT) and various
special cases thereof. It is an open-source implementation of the Fast
Fourier transform algorithm. It can compute transforms of real and
complex-values arrays of arbitrary size and dimension. AMD Optimized
FFTW is the optimized FFTW implementation targeted for AMD CPUs. For
single precision build, please use precision value as float. Example :
spack install amdfftw precision=float</p>
<p><strong>Versions</strong>: 3.2</p>
<p><strong>Arches</strong>: amd</p>
<p><strong>Modules</strong>: <code>amdfftw/3.2+amd</code></p>
<h2 id="ant">ant<a class="headerlink" href="#ant" title="Permanent link">&para;</a></h2>
<p>Apache Ant is a Java library and command-line tool whose mission is to
drive processes described in build files as targets and extension points
dependent upon each other</p>
<p><strong>Versions</strong>: 1.10.7</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ant/1.10.7</code></p>
<h2 id="aragorn">aragorn<a class="headerlink" href="#aragorn" title="Permanent link">&para;</a></h2>
<p>ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide
sequences.</p>
<p><strong>Versions</strong>: 1.2.38</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>aragorn/1.2.38</code></p>
<h2 id="bedtools2">bedtools2<a class="headerlink" href="#bedtools2" title="Permanent link">&para;</a></h2>
<p>Collectively, the bedtools utilities are a swiss-army knife of tools for
a wide-range of genomics analysis tasks. The most widely-used tools
enable genome arithmetic: that is, set theory on the genome.</p>
<p><strong>Versions</strong>: 2.30.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>bedtools2/2.30.0</code></p>
<h2 id="blast-plus">blast-plus<a class="headerlink" href="#blast-plus" title="Permanent link">&para;</a></h2>
<p>Basic Local Alignment Search Tool.</p>
<p><strong>Versions</strong>: 2.12.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>blast-plus/2.12.0</code></p>
<h2 id="blast2go">blast2go<a class="headerlink" href="#blast2go" title="Permanent link">&para;</a></h2>
<p>Blast2GO is a bioinformatics platform for high-quality functional
annotation and analysis of genomic datasets.</p>
<p><strong>Versions</strong>: 5.2.5</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>blast2go/5.2.5</code></p>
<h2 id="blat">blat<a class="headerlink" href="#blat" title="Permanent link">&para;</a></h2>
<p>BLAT (BLAST-like alignment tool) is a pairwise sequence alignment
algorithm.</p>
<p><strong>Versions</strong>: 35</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>blat/35</code></p>
<h2 id="bowtie">bowtie<a class="headerlink" href="#bowtie" title="Permanent link">&para;</a></h2>
<p>Bowtie is an ultrafast, memory-efficient short read aligner for short
DNA sequences (reads) from next-gen sequencers.</p>
<p><strong>Versions</strong>: 1.3.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>bowtie/1.3.0</code></p>
<h2 id="bowtie2">bowtie2<a class="headerlink" href="#bowtie2" title="Permanent link">&para;</a></h2>
<p>Bowtie 2 is an ultrafast and memory-efficient tool for aligning
sequencing reads to long reference sequences</p>
<p><strong>Versions</strong>: 2.4.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>bowtie2/2.4.2</code></p>
<h2 id="bwa">bwa<a class="headerlink" href="#bwa" title="Permanent link">&para;</a></h2>
<p>Burrow-Wheeler Aligner for pairwise alignment between DNA sequences.</p>
<p><strong>Versions</strong>: 0.7.17</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>bwa/0.7.17</code></p>
<h2 id="bwtool">bwtool<a class="headerlink" href="#bwtool" title="Permanent link">&para;</a></h2>
<p>bwtool is a command-line utility for bigWig files.</p>
<p><strong>Versions</strong>: 1.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>bwtool/1.0</code></p>
<h2 id="canu">canu<a class="headerlink" href="#canu" title="Permanent link">&para;</a></h2>
<p>A single molecule sequence assembler for genomes large and small.</p>
<p><strong>Versions</strong>: 2.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>canu/2.2</code></p>
<h2 id="cap3">cap3<a class="headerlink" href="#cap3" title="Permanent link">&para;</a></h2>
<p>CAP3 is DNA Sequence Assembly Program</p>
<p><strong>Versions</strong>: 2015-02-11</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>cap3/2015-02-11</code></p>
<h2 id="clustal-omega">clustal-omega<a class="headerlink" href="#clustal-omega" title="Permanent link">&para;</a></h2>
<p>Clustal Omega: the last alignment program you'll ever need.</p>
<p><strong>Versions</strong>: 1.2.4</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>clustal-omega/1.2.4</code></p>
<h2 id="clustalw">clustalw<a class="headerlink" href="#clustalw" title="Permanent link">&para;</a></h2>
<p>Multiple alignment of nucleic acid and protein sequences.</p>
<p><strong>Versions</strong>: 2.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>clustalw/2.1</code></p>
<h2 id="corset">corset<a class="headerlink" href="#corset" title="Permanent link">&para;</a></h2>
<p>Corset is a command-line software program to go from a de novo
transcriptome assembly to gene-level counts.</p>
<p><strong>Versions</strong>: 1.09</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>corset/1.09</code></p>
<h2 id="ctffind">ctffind<a class="headerlink" href="#ctffind" title="Permanent link">&para;</a></h2>
<p>Fast and accurate defocus estimation from electron micrographs.</p>
<p><strong>Versions</strong>: 4.1.14</p>
<p><strong>Arches</strong>: amd, intel</p>
<p><strong>Modules</strong>: <code>ctffind/4.1.14+intel</code>, <code>ctffind/4.1.14+amd</code></p>
<h2 id="cuda">cuda<a class="headerlink" href="#cuda" title="Permanent link">&para;</a></h2>
<p>CUDA is a parallel computing platform and programming model invented by
NVIDIA. It enables dramatic increases in computing performance by
harnessing the power of the graphics processing unit (GPU). Note: This
package does not currently install the drivers necessary to run CUDA.
These will need to be installed manually. See:
https://docs.nvidia.com/cuda/ for details.</p>
<p><strong>Versions</strong>: 11.7.1, 11.8.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>cuda/11.8.0</code>, <code>cuda/11.7.1</code></p>
<h2 id="cufflinks">cufflinks<a class="headerlink" href="#cufflinks" title="Permanent link">&para;</a></h2>
<p>Cufflinks assembles transcripts, estimates their abundances, and tests
for differential expression and regulation in RNA-Seq samples.</p>
<p><strong>Versions</strong>: 2.2.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>cufflinks/2.2.1</code></p>
<h2 id="ea-utils">ea-utils<a class="headerlink" href="#ea-utils" title="Permanent link">&para;</a></h2>
<p>Command-line tools for processing biological sequencing data. Barcode
demultiplexing, adapter trimming, etc. Primarily written to support an
Illumina based pipeline - but should work with any FASTQs.</p>
<p><strong>Versions</strong>: 2021-10-20</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ea-utils/2021-10-20</code></p>
<h2 id="emboss">emboss<a class="headerlink" href="#emboss" title="Permanent link">&para;</a></h2>
<p>EMBOSS is a free Open Source software analysis package specially
developed for the needs of the molecular biology (e.g. EMBnet) user
community</p>
<p><strong>Versions</strong>: 6.6.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>emboss/6.6.0</code></p>
<h2 id="exonerate">exonerate<a class="headerlink" href="#exonerate" title="Permanent link">&para;</a></h2>
<p>Pairwise sequence alignment of DNA and proteins</p>
<p><strong>Versions</strong>: 2.4.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>exonerate/2.4.0</code></p>
<h2 id="exonerate-gff3">exonerate-gff3<a class="headerlink" href="#exonerate-gff3" title="Permanent link">&para;</a></h2>
<p>This is an exonerate fork with added gff3 support. Original website with
user guides: http://www.ebi.ac.uk/~guy/exonerate/</p>
<p><strong>Versions</strong>: 2.3.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>exonerate-gff3/2.3.0</code></p>
<h2 id="fastqc">fastqc<a class="headerlink" href="#fastqc" title="Permanent link">&para;</a></h2>
<p>A quality control tool for high throughput sequence data.</p>
<p><strong>Versions</strong>: 0.11.9</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>fastqc/0.11.9</code></p>
<h2 id="fftw">fftw<a class="headerlink" href="#fftw" title="Permanent link">&para;</a></h2>
<p>FFTW is a C subroutine library for computing the discrete Fourier
transform (DFT) in one or more dimensions, of arbitrary input size, and
of both real and complex data (as well as of even/odd data, i.e. the
discrete cosine/sine transforms or DCT/DST). We believe that FFTW, which
is free software, should become the FFT library of choice for most
applications.</p>
<p><strong>Versions</strong>: 3.3.10</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>fftw/3.3.10</code></p>
<h2 id="freebayes">freebayes<a class="headerlink" href="#freebayes" title="Permanent link">&para;</a></h2>
<p>Bayesian haplotype-based genetic polymorphism discovery and genotyping.</p>
<p><strong>Versions</strong>: 1.3.6</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>freebayes/1.3.6</code></p>
<h2 id="gatk">gatk<a class="headerlink" href="#gatk" title="Permanent link">&para;</a></h2>
<p>Genome Analysis Toolkit Variant Discovery in High-Throughput Sequencing
Data</p>
<p><strong>Versions</strong>: 4.2.6.1, 3.8.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>gatk/3.8.1</code>, <code>gatk/4.2.6.1</code></p>
<h2 id="gcc">gcc<a class="headerlink" href="#gcc" title="Permanent link">&para;</a></h2>
<p>The GNU Compiler Collection includes front ends for C, C++, Objective-C,
Fortran, Ada, and Go, as well as libraries for these languages.</p>
<p><strong>Versions</strong>: 4.9.4, 5.5.0, 7.5.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>gcc/5.5.0</code>, <code>gcc/7.5.0</code>, <code>gcc/4.9.4</code></p>
<h2 id="gctf">gctf<a class="headerlink" href="#gctf" title="Permanent link">&para;</a></h2>
<p>a GPU accelerated program for Real-Time CTF determination, refinement,
evaluation and correction.</p>
<p><strong>Versions</strong>: 1.06</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>gctf/1.06</code></p>
<h2 id="genrich">genrich<a class="headerlink" href="#genrich" title="Permanent link">&para;</a></h2>
<p>Genrich is a peak-caller for genomic enrichment assays.</p>
<p><strong>Versions</strong>: 0.6</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>genrich/0.6</code></p>
<h2 id="ghostscript">ghostscript<a class="headerlink" href="#ghostscript" title="Permanent link">&para;</a></h2>
<p>An interpreter for the PostScript language and for PDF.</p>
<p><strong>Versions</strong>: 9.56.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ghostscript/9.56.1</code></p>
<h2 id="glimmer">glimmer<a class="headerlink" href="#glimmer" title="Permanent link">&para;</a></h2>
<p>Glimmer is a system for finding genes in microbial DNA, especially the
genomes of bacteria, archaea, and viruses.</p>
<p><strong>Versions</strong>: 3.02b</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>glimmer/3.02b</code></p>
<h2 id="hdf5">hdf5<a class="headerlink" href="#hdf5" title="Permanent link">&para;</a></h2>
<p>HDF5 is a data model, library, and file format for storing and managing
data. It supports an unlimited variety of datatypes, and is designed for
flexible and efficient I/O and for high volume and complex data.</p>
<p><strong>Versions</strong>: 1.12.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>hdf5/1.12.2</code></p>
<h2 id="hisat2">hisat2<a class="headerlink" href="#hisat2" title="Permanent link">&para;</a></h2>
<p>HISAT2 is a fast and sensitive alignment program for mapping next-
generation sequencing reads (whole-genome, transcriptome, and exome
sequencing data) against the general human population (as well as
against a single reference genome).</p>
<p><strong>Versions</strong>: 2.2.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>hisat2/2.2.0</code></p>
<h2 id="hmmer">hmmer<a class="headerlink" href="#hmmer" title="Permanent link">&para;</a></h2>
<p>HMMER is used for searching sequence databases for sequence homologs,
and for making sequence alignments. It implements methods using
probabilistic models called profile hidden Markov models (profile HMMs).</p>
<p><strong>Versions</strong>: 3.3.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>hmmer/3.3.2</code></p>
<h2 id="homer">homer<a class="headerlink" href="#homer" title="Permanent link">&para;</a></h2>
<p>Software for motif discovery and next generation sequencing analysis</p>
<p><strong>Versions</strong>: 4.9.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>homer/4.9.1</code></p>
<h2 id="hwloc">hwloc<a class="headerlink" href="#hwloc" title="Permanent link">&para;</a></h2>
<p>The Hardware Locality (hwloc) software project. The Portable Hardware
Locality (hwloc) software package provides a portable abstraction
(across OS, versions, architectures, ...) of the hierarchical topology
of modern architectures, including NUMA memory nodes, sockets, shared
caches, cores and simultaneous multithreading. It also gathers various
system attributes such as cache and memory information as well as the
locality of I/O devices such as network interfaces, InfiniBand HCAs or
GPUs. It primarily aims at helping applications with gathering
information about modern computing hardware so as to exploit it
accordingly and efficiently.</p>
<p><strong>Versions</strong>: 2.8.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>hwloc/2.8.0</code></p>
<h2 id="igv">igv<a class="headerlink" href="#igv" title="Permanent link">&para;</a></h2>
<p>The Integrative Genomics Viewer (IGV) is a high-performance
visualization tool for interactive exploration of large, integrated
genomic datasets. It supports a wide variety of data types, including
array-based and next-generation sequence data, and genomic annotations.</p>
<p><strong>Versions</strong>: 2.12.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>igv/2.12.3</code></p>
<h2 id="infernal">infernal<a class="headerlink" href="#infernal" title="Permanent link">&para;</a></h2>
<p>Infernal (INFERence of RNA ALignment) is for searching DNA sequence
databases for RNA structure and sequence similarities. It is an
implementation of a special case of profile stochastic context-free
grammars called covariance models (CMs).</p>
<p><strong>Versions</strong>: 1.1.4</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>infernal/1.1.4</code></p>
<h2 id="intel-oneapi-compilers">intel-oneapi-compilers<a class="headerlink" href="#intel-oneapi-compilers" title="Permanent link">&para;</a></h2>
<p>Intel oneAPI Compilers. Includes: icc, icpc, ifort, icx, icpx, ifx, and
dpcpp. LICENSE INFORMATION: By downloading and using this software, you
agree to the terms and conditions of the software license agreements at
https://intel.ly/393CijO.</p>
<p><strong>Versions</strong>: 2022.2.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>intel-oneapi-compilers/2022.2.1</code></p>
<h2 id="intel-oneapi-mkl">intel-oneapi-mkl<a class="headerlink" href="#intel-oneapi-mkl" title="Permanent link">&para;</a></h2>
<p>Intel oneAPI Math Kernel Library (Intel oneMKL; formerly Intel Math
Kernel Library or Intel MKL), is a library of optimized math routines
for science, engineering, and financial applications. Core math
functions include BLAS, LAPACK, ScaLAPACK, sparse solvers, fast Fourier
transforms, and vector math. LICENSE INFORMATION: By downloading and
using this software, you agree to the terms and conditions of the
software license agreements at https://intel.ly/393CijO.</p>
<p><strong>Versions</strong>: 2022.2.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>intel-oneapi-mkl/2022.2.1</code></p>
<h2 id="interproscan">interproscan<a class="headerlink" href="#interproscan" title="Permanent link">&para;</a></h2>
<p>InterProScan is the software package that allows sequences (protein and
nucleic) to be scanned against InterPro's signatures. Signatures are
predictive models, provided by several different databases, that make up
the InterPro consortium.</p>
<p><strong>Versions</strong>: 5.56-89.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>interproscan/5.56-89.0</code></p>
<h2 id="iq-tree">iq-tree<a class="headerlink" href="#iq-tree" title="Permanent link">&para;</a></h2>
<p>IQ-TREE Efficient software for phylogenomic inference</p>
<p><strong>Versions</strong>: 2.1.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>iq-tree/2.1.3</code></p>
<h2 id="iqtree2">iqtree2<a class="headerlink" href="#iqtree2" title="Permanent link">&para;</a></h2>
<p>Efficient and versatile phylogenomic software by maximum likelihood</p>
<p><strong>Versions</strong>: 2.1.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>iqtree2/2.1.2</code></p>
<h2 id="jdk">jdk<a class="headerlink" href="#jdk" title="Permanent link">&para;</a></h2>
<p>The Java Development Kit (JDK) released by Oracle Corporation in the
form of a binary product aimed at Java developers. Includes a complete
JRE plus tools for developing, debugging, and monitoring Java
applications.</p>
<p><strong>Versions</strong>: 17.0.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>jdk/17.0.1</code></p>
<h2 id="jellyfish">jellyfish<a class="headerlink" href="#jellyfish" title="Permanent link">&para;</a></h2>
<p>JELLYFISH is a tool for fast, memory-efficient counting of k-mers in
DNA.</p>
<p><strong>Versions</strong>: 1.1.11</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>jellyfish/1.1.11</code></p>
<h2 id="kalign">kalign<a class="headerlink" href="#kalign" title="Permanent link">&para;</a></h2>
<p>A fast multiple sequence alignment program for biological sequences.</p>
<p><strong>Versions</strong>: 3.3.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>kalign/3.3.1</code></p>
<h2 id="kallisto">kallisto<a class="headerlink" href="#kallisto" title="Permanent link">&para;</a></h2>
<p>kallisto is a program for quantifying abundances of transcripts from
RNA-Seq data.</p>
<p><strong>Versions</strong>: 0.48.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>kallisto/0.48.0</code></p>
<h2 id="kmergenie">kmergenie<a class="headerlink" href="#kmergenie" title="Permanent link">&para;</a></h2>
<p>KmerGenie estimates the best k-mer length for genome de novo assembly.</p>
<p><strong>Versions</strong>: 1.7044</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>kmergenie/1.7044</code></p>
<h2 id="kraken">kraken<a class="headerlink" href="#kraken" title="Permanent link">&para;</a></h2>
<p>Kraken is a system for assigning taxonomic labels to short DNA
sequences, usually obtained through metagenomic studies.</p>
<p><strong>Versions</strong>: 1.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>kraken/1.0</code></p>
<h2 id="kraken2">kraken2<a class="headerlink" href="#kraken2" title="Permanent link">&para;</a></h2>
<p>Kraken2 is a system for assigning taxonomic labels to short DNA
sequences, usually obtained through metagenomic studies.</p>
<p><strong>Versions</strong>: 2.1.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>kraken2/2.1.2</code></p>
<h2 id="krakenuniq">krakenuniq<a class="headerlink" href="#krakenuniq" title="Permanent link">&para;</a></h2>
<p>Metagenomics classifier with unique k-mer counting for more specific
results.</p>
<p><strong>Versions</strong>: 0.7.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>krakenuniq/0.7.3</code></p>
<h2 id="last">last<a class="headerlink" href="#last" title="Permanent link">&para;</a></h2>
<p>LAST finds similar regions between sequences, and aligns them. It is
designed for comparing large datasets to each other (e.g. vertebrate
genomes and/or large numbers of DNA reads).</p>
<p><strong>Versions</strong>: 1282</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>last/1282</code></p>
<h2 id="libevent">libevent<a class="headerlink" href="#libevent" title="Permanent link">&para;</a></h2>
<p>The libevent API provides a mechanism to execute a callback function
when a specific event occurs on a file descriptor or after a timeout has
been reached. Furthermore, libevent also support callbacks due to
signals or regular timeouts.</p>
<p><strong>Versions</strong>: 2.1.12</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>libevent/2.1.12</code></p>
<h2 id="mash">mash<a class="headerlink" href="#mash" title="Permanent link">&para;</a></h2>
<p>Fast genome and metagenome distance estimation using MinHash.</p>
<p><strong>Versions</strong>: 2.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mash/2.3</code></p>
<h2 id="masurca">masurca<a class="headerlink" href="#masurca" title="Permanent link">&para;</a></h2>
<p>MaSuRCA is whole genome assembly software. It combines the efficiency of
the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches.</p>
<p><strong>Versions</strong>: 4.0.9</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>masurca/4.0.9</code></p>
<h2 id="mcl">mcl<a class="headerlink" href="#mcl" title="Permanent link">&para;</a></h2>
<p>The MCL algorithm is short for the Markov Cluster Algorithm, a fast and
scalable unsupervised cluster algorithm for graphs (also known as
networks) based on simulation of (stochastic) flow in graphs.</p>
<p><strong>Versions</strong>: 14-137</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mcl/14-137</code></p>
<h2 id="megahit">megahit<a class="headerlink" href="#megahit" title="Permanent link">&para;</a></h2>
<p>MEGAHIT: An ultra-fast single-node solution for large and complex
metagenomics assembly via succinct de Bruijn graph</p>
<p><strong>Versions</strong>: 1.1.4</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>megahit/1.1.4</code></p>
<h2 id="meme">meme<a class="headerlink" href="#meme" title="Permanent link">&para;</a></h2>
<p>The MEME Suite allows the biologist to discover novel motifs in
collections of unaligned nucleotide or protein sequences, and to perform
a wide variety of other motif-based analyses.</p>
<p><strong>Versions</strong>: 5.3.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>meme/5.3.0</code></p>
<h2 id="metaeuk">metaeuk<a class="headerlink" href="#metaeuk" title="Permanent link">&para;</a></h2>
<p>MetaEuk is a modular toolkit designed for large-scale gene discovery and
annotation in eukaryotic metagenomic contigs.</p>
<p><strong>Versions</strong>: 6-a5d39d9</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>metaeuk/6-a5d39d9</code></p>
<h2 id="minced">minced<a class="headerlink" href="#minced" title="Permanent link">&para;</a></h2>
<p>MinCED is a program to find Clustered Regularly Interspaced Short
Palindromic Repeats (CRISPRs) in full genomes or environmental datasets
such as metagenomes, in which sequence size can be anywhere from 100 to
800 bp.</p>
<p><strong>Versions</strong>: 0.3.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>minced/0.3.2</code></p>
<h2 id="miniasm">miniasm<a class="headerlink" href="#miniasm" title="Permanent link">&para;</a></h2>
<p>Miniasm is a very fast OLC-based de novo assembler for noisy long reads.</p>
<p><strong>Versions</strong>: 2018-3-30</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>miniasm/2018-3-30</code></p>
<h2 id="minimap2">minimap2<a class="headerlink" href="#minimap2" title="Permanent link">&para;</a></h2>
<p>Minimap2 is a versatile sequence alignment program that aligns DNA or
mRNA sequences against a large reference database. Mappy provides a
convenient interface to minimap2.</p>
<p><strong>Versions</strong>: 2.14</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>minimap2/2.14</code></p>
<h2 id="mirdeep2">mirdeep2<a class="headerlink" href="#mirdeep2" title="Permanent link">&para;</a></h2>
<p>miRDeep2 is a completely overhauled tool which discovers microRNA genes
by analyzing sequenced RNAs.</p>
<p><strong>Versions</strong>: 0.0.8</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mirdeep2/0.0.8</code></p>
<h2 id="mmseqs2">mmseqs2<a class="headerlink" href="#mmseqs2" title="Permanent link">&para;</a></h2>
<p>MMseqs2 (Many-against-Many sequence searching) is a software suite to
search and cluster huge protein and nucleotide sequence sets</p>
<p><strong>Versions</strong>: 14-7e284</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mmseqs2/14-7e284</code></p>
<h2 id="mothur">mothur<a class="headerlink" href="#mothur" title="Permanent link">&para;</a></h2>
<p>This project seeks to develop a single piece of open-source, expandable
software to fill the bioinformatics needs of the microbial ecology
community.</p>
<p><strong>Versions</strong>: 1.48.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mothur/1.48.0</code></p>
<h2 id="motioncor2">motioncor2<a class="headerlink" href="#motioncor2" title="Permanent link">&para;</a></h2>
<p>MotionCor2 is a multi-GPU program that corrects beam-induced sample
motion recorded on dose fractionated movie stacks. It implements a
robust iterative alignment algorithm that delivers precise measurement
and correction of both global and non-uniform local motions at single
pixel level, suitable for both single-particle and tomographic images.
MotionCor2 is sufficiently fast to keep up with automated data
collection.</p>
<p><strong>Versions</strong>: 1.5.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>motioncor2/1.5.0</code></p>
<h2 id="mummer">mummer<a class="headerlink" href="#mummer" title="Permanent link">&para;</a></h2>
<p>MUMmer is a system for rapidly aligning entire genomes.</p>
<p><strong>Versions</strong>: 3.23</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mummer/3.23</code></p>
<h2 id="mummer4">mummer4<a class="headerlink" href="#mummer4" title="Permanent link">&para;</a></h2>
<p>MUMmer is a versatil alignment tool for DNA and protein sequences.</p>
<p><strong>Versions</strong>: 4.0.0rc1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>mummer4/4.0.0rc1</code></p>
<h2 id="muscle">muscle<a class="headerlink" href="#muscle" title="Permanent link">&para;</a></h2>
<p>MUSCLE is one of the best-performing multiple alignment programs
according to published benchmark tests, with accuracy and speed that are
consistently better than CLUSTALW.</p>
<p><strong>Versions</strong>: 3.8.1551</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>muscle/3.8.1551</code></p>
<h2 id="ncbi-rmblastn">ncbi-rmblastn<a class="headerlink" href="#ncbi-rmblastn" title="Permanent link">&para;</a></h2>
<p>RMBlast search engine for NCBI</p>
<p><strong>Versions</strong>: 2.11.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ncbi-rmblastn/2.11.0</code></p>
<h2 id="ncbi-toolkit">ncbi-toolkit<a class="headerlink" href="#ncbi-toolkit" title="Permanent link">&para;</a></h2>
<p>NCBI C++ Toolkit</p>
<p><strong>Versions</strong>: 26_0_1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ncbi-toolkit/26_0_1</code></p>
<h2 id="ncbi-vdb">ncbi-vdb<a class="headerlink" href="#ncbi-vdb" title="Permanent link">&para;</a></h2>
<p>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries
for using data in the INSDC Sequence Read Archives. This package
contains the interface to the VDB.</p>
<p><strong>Versions</strong>: 3.0.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ncbi-vdb/3.0.0</code></p>
<h2 id="nextflow">nextflow<a class="headerlink" href="#nextflow" title="Permanent link">&para;</a></h2>
<p>Data-driven computational pipelines.</p>
<p><strong>Versions</strong>: 22.10.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>nextflow/22.10.1</code></p>
<h2 id="openjdk">openjdk<a class="headerlink" href="#openjdk" title="Permanent link">&para;</a></h2>
<p>The free and opensource java implementation</p>
<p><strong>Versions</strong>: 11.0.17_8, 16.0.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>openjdk/11.0.17_8</code>, <code>openjdk/16.0.2</code></p>
<h2 id="openldap">openldap<a class="headerlink" href="#openldap" title="Permanent link">&para;</a></h2>
<p>OpenLDAP Software is an open source implementation of the Lightweight
Directory Access Protocol. The suite includes: slapd - stand-alone LDAP
daemon (server) libraries implementing the LDAP protocol, and utilities,
tools, and sample clients.</p>
<p><strong>Versions</strong>: 2.4.49</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>openldap/2.4.49</code></p>
<h2 id="openmpi">openmpi<a class="headerlink" href="#openmpi" title="Permanent link">&para;</a></h2>
<p>An open source Message Passing Interface implementation. The Open MPI
Project is an open source Message Passing Interface implementation that
is developed and maintained by a consortium of academic, research, and
industry partners. Open MPI is therefore able to combine the expertise,
technologies, and resources from all across the High Performance
Computing community in order to build the best MPI library available.
Open MPI offers advantages for system and software vendors, application
developers and computer science researchers.</p>
<p><strong>Versions</strong>: 4.1.4</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>openmpi/4.1.4</code></p>
<h2 id="orthofinder">orthofinder<a class="headerlink" href="#orthofinder" title="Permanent link">&para;</a></h2>
<p>OrthoFinder is a fast, accurate and comprehensive analysis tool for
comparative genomics. It finds orthologues and orthogroups infers rooted
gene trees for all orthogroups and infers a rooted species tree for the
species being analysed. OrthoFinder also provides comprehensive
statistics for comparative genomic analyses. OrthoFinder is simple to
use and all you need to run it is a set of protein sequence files (one
per species) in FASTA format.</p>
<p><strong>Versions</strong>: 2.5.4</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>orthofinder/2.5.4</code></p>
<h2 id="orthomcl">orthomcl<a class="headerlink" href="#orthomcl" title="Permanent link">&para;</a></h2>
<p>OrthoMCL is a genome-scale algorithm for grouping orthologous protein
sequences.</p>
<p><strong>Versions</strong>: 2.0.9</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>orthomcl/2.0.9</code></p>
<h2 id="parallel">parallel<a class="headerlink" href="#parallel" title="Permanent link">&para;</a></h2>
<p>GNU parallel is a shell tool for executing jobs in parallel using one or
more computers. A job can be a single command or a small script that has
to be run for each of the lines in the input.</p>
<p><strong>Versions</strong>: 20220522</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>parallel/20220522</code></p>
<h2 id="patchelf">patchelf<a class="headerlink" href="#patchelf" title="Permanent link">&para;</a></h2>
<p>PatchELF is a small utility to modify the dynamic linker and RPATH of
ELF executables.</p>
<p><strong>Versions</strong>: 0.16.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>patchelf/0.16.1</code></p>
<h2 id="phylip">phylip<a class="headerlink" href="#phylip" title="Permanent link">&para;</a></h2>
<p>PHYLIP (the PHYLogeny Inference Package) is a package of programs for
inferring phylogenies (evolutionary trees).</p>
<p><strong>Versions</strong>: 3.697</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>phylip/3.697</code></p>
<h2 id="picard">picard<a class="headerlink" href="#picard" title="Permanent link">&para;</a></h2>
<p>Picard is a set of command line tools for manipulating high-throughput
sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.</p>
<p><strong>Versions</strong>: 2.26.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>picard/2.26.2</code></p>
<h2 id="pilon">pilon<a class="headerlink" href="#pilon" title="Permanent link">&para;</a></h2>
<p>Pilon is an automated genome assembly improvement and variant detection
tool.</p>
<p><strong>Versions</strong>: 1.22</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>pilon/1.22</code></p>
<h2 id="plink">plink<a class="headerlink" href="#plink" title="Permanent link">&para;</a></h2>
<p>PLINK is a free, open-source whole genome association analysis toolset,
designed to perform a range of basic, large-scale analyses in a
computationally efficient manner.</p>
<p><strong>Versions</strong>: 1.07</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>plink/1.07</code></p>
<h2 id="pmix">pmix<a class="headerlink" href="#pmix" title="Permanent link">&para;</a></h2>
<p>The Process Management Interface (PMI) has been used for quite some time
as a means of exchanging wireup information needed for interprocess
communication. However, meeting the significant orchestration challenges
presented by exascale systems requires that the process-to-system
interface evolve to permit a tighter integration between the different
components of the parallel application and existing and future SMS
solutions. PMI Exascale (PMIx) addresses these needs by providing an
extended version of the PMI definitions specifically designed to support
exascale and beyond environments by: (a) adding flexibility to the
functionality expressed in the existing APIs, (b) augmenting the
interfaces with new APIs that provide extended capabilities, (c) forging
a collaboration between subsystem providers including resource manager,
fabric, file system, and programming library developers, (d)
establishing a standards-like body for maintaining the definitions, and
(e) providing a reference implementation of the PMIx standard that
demonstrates the desired level of scalability while maintaining strict
separation between it and the standard itself.</p>
<p><strong>Versions</strong>: 4.1.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>pmix/4.1.2</code></p>
<h2 id="prokka">prokka<a class="headerlink" href="#prokka" title="Permanent link">&para;</a></h2>
<p>Prokka is a software tool to annotate bacterial, archaeal and viral
genomes quickly and produce standards-compliant output files.</p>
<p><strong>Versions</strong>: 1.14.6</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>prokka/1.14.6</code></p>
<h2 id="r_1">r<a class="headerlink" href="#r_1" title="Permanent link">&para;</a></h2>
<p>R is 'GNU S', a freely available language and environment for
statistical computing and graphics which provides a wide variety of
statistical and graphical techniques: linear and nonlinear modelling,
statistical tests, time series analysis, classification, clustering,
etc. Please consult the R project homepage for further information.</p>
<p><strong>Versions</strong>: 4.2.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>r/4.2.0</code></p>
<h2 id="raxml-ng">raxml-ng<a class="headerlink" href="#raxml-ng" title="Permanent link">&para;</a></h2>
<p>RAxML-NG is a phylogenetic tree inference tool which uses maximum-
likelihood (ML) optimality criterion. Its search heuristic is based on
iteratively performing a series of Subtree Pruning and Regrafting (SPR)
moves, which allows to quickly navigate to the best-known ML tree.
RAxML-NG is a successor of RAxML (Stamatakis 2014) and leverages the
highly optimized likelihood computation implemented in libpll (Flouri et
al. 2014).</p>
<p><strong>Versions</strong>: 1.0.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>raxml-ng/1.0.2</code></p>
<h2 id="ray">ray<a class="headerlink" href="#ray" title="Permanent link">&para;</a></h2>
<p>Parallel genome assemblies for parallel DNA sequencing</p>
<p><strong>Versions</strong>: 2.3.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ray/2.3.1</code></p>
<h2 id="rclone">rclone<a class="headerlink" href="#rclone" title="Permanent link">&para;</a></h2>
<p>Rclone is a command line program to sync files and directories to and
from various cloud storage providers</p>
<p><strong>Versions</strong>: 1.59.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>rclone/1.59.1</code></p>
<h2 id="recon">recon<a class="headerlink" href="#recon" title="Permanent link">&para;</a></h2>
<p>RECON: a package for automated de novo identification of repeat families
from genomic sequences.</p>
<p><strong>Versions</strong>: 1.05</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>recon/1.05</code></p>
<h2 id="relion">relion<a class="headerlink" href="#relion" title="Permanent link">&para;</a></h2>
<p>RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a
stand-alone computer program that employs an empirical Bayesian approach
to refinement of (multiple) 3D reconstructions or 2D class averages in
electron cryo-microscopy (cryo-EM).</p>
<p><strong>Versions</strong>: 4.0.1, 3.1.3, 4.0.0</p>
<p><strong>Variants</strong>: cpu, gpu</p>
<p><strong>Arches</strong>: amd, intel, intel+gpu_delay</p>
<p><strong>Modules</strong>: <code>relion/cpu/3.1.3+amd</code>, <code>relion/gpu/3.1.3+amd</code>, <code>relion/gpu/3.1.3+intel</code>, <code>relion/cpu/4.0.0+amd</code>, <code>relion/gpu/4.0.0+amd</code>, <code>relion/gpu/4.0.0+intel</code>, <code>relion/3.1.3</code>, <code>relion/4.0.0</code>, <code>relion/cpu/4.0.1+amd</code>, <code>relion/gpu/4.0.1+amd</code>, <code>relion/4.0.1</code>, <code>relion/gpu/4.0.1+intel+gpu_delay</code>, <code>relion/gpu/4.0.1+intel</code></p>
<h2 id="relion-bbr">relion-bbr<a class="headerlink" href="#relion-bbr" title="Permanent link">&para;</a></h2>
<p>A modified version of Relion supporting block-based-reconstruction as
described in 10.1038/s41467-018-04051-9.</p>
<p><strong>Versions</strong>: 3.1.2</p>
<p><strong>Variants</strong>: gpu</p>
<p><strong>Arches</strong>: intel</p>
<p><strong>Modules</strong>: <code>relion-bbr/gpu/3.1.2+intel</code></p>
<h2 id="relion-helper">relion-helper<a class="headerlink" href="#relion-helper" title="Permanent link">&para;</a></h2>
<p>Utilities for Relion Cryo-EM data processing on clusters.</p>
<p><strong>Versions</strong>: 0.2, 0.1, 0.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>relion-helper/0.1</code>, <code>relion-helper/0.2</code>, <code>relion-helper/0.3</code></p>
<h2 id="repeatmasker">repeatmasker<a class="headerlink" href="#repeatmasker" title="Permanent link">&para;</a></h2>
<p>RepeatMasker is a program that screens DNA sequences for interspersed
repeats and low complexity DNA sequences.</p>
<p><strong>Versions</strong>: 4.0.9</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>repeatmasker/4.0.9</code></p>
<h2 id="repeatmodeler">repeatmodeler<a class="headerlink" href="#repeatmodeler" title="Permanent link">&para;</a></h2>
<p>RepeatModeler is a de-novo repeat family identification and modeling
package.</p>
<p><strong>Versions</strong>: 1.0.11</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>repeatmodeler/1.0.11</code></p>
<h2 id="repeatscout">repeatscout<a class="headerlink" href="#repeatscout" title="Permanent link">&para;</a></h2>
<p>RepeatScout - De Novo Repeat Finder, Price A.L., Jones N.C. and Pevzner
P.A.</p>
<p><strong>Versions</strong>: 1.0.5</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>repeatscout/1.0.5</code></p>
<h2 id="rnaquast">rnaquast<a class="headerlink" href="#rnaquast" title="Permanent link">&para;</a></h2>
<p>Quality assessment of de novo transcriptome assemblies from RNA-Seq data
rnaQUAST is a tool for evaluating RNA-Seq assemblies using reference
genome and gene database. In addition, rnaQUAST is also capable of
estimating gene database coverage by raw reads and de novo quality
assessment using third-party software.</p>
<p><strong>Versions</strong>: 2.2.0</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>rnaquast/2.2.0</code></p>
<h2 id="rsem">rsem<a class="headerlink" href="#rsem" title="Permanent link">&para;</a></h2>
<p>RSEM is a software package for estimating gene and isoform expression
levels from RNA-Seq data.</p>
<p><strong>Versions</strong>: 1.3.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>rsem/1.3.1</code></p>
<h2 id="rstudio-server">rstudio-server<a class="headerlink" href="#rstudio-server" title="Permanent link">&para;</a></h2>
<p>RStudio is an integrated development environment (IDE) for R.</p>
<p><strong>Versions</strong>: 2022.12.0-353</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>rstudio-server/2022.12.0-353</code></p>
<h2 id="sabre">sabre<a class="headerlink" href="#sabre" title="Permanent link">&para;</a></h2>
<p>Sabre is a tool that will demultiplex barcoded reads into separate
files. It will work on both single-end and paired-end data in fastq
format. It simply compares the provided barcodes with each read and
separates the read into its appropriate barcode file, after stripping
the barcode from the read (and also stripping the quality values of the
barcode bases). If a read does not have a recognized barcode, then it is
put into the unknown file.</p>
<p><strong>Versions</strong>: 2013-09-27</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>sabre/2013-09-27</code></p>
<h2 id="satsuma2">satsuma2<a class="headerlink" href="#satsuma2" title="Permanent link">&para;</a></h2>
<p>Satsuma2 is an optimsed version of Satsuma, a tool to reliably align
large and complex DNA sequences providing maximum sensitivity (to find
all there is to find), specificity (to only find real homology) and
speed (to accomodate the billions of base pairs in vertebrate genomes).</p>
<p><strong>Versions</strong>: 2021-03-04</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>satsuma2/2021-03-04</code></p>
<h2 id="scallop">scallop<a class="headerlink" href="#scallop" title="Permanent link">&para;</a></h2>
<p>Scallop is a reference-based transcriptome assembler for RNA-seq</p>
<p><strong>Versions</strong>: 0.10.5</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>scallop/0.10.5</code></p>
<h2 id="seqprep">seqprep<a class="headerlink" href="#seqprep" title="Permanent link">&para;</a></h2>
<p>SeqPrep is a program to merge paired end Illumina reads that are
overlapping into a single longer read.</p>
<p><strong>Versions</strong>: 1.3.2</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>seqprep/1.3.2</code></p>
<h2 id="seqtk">seqtk<a class="headerlink" href="#seqtk" title="Permanent link">&para;</a></h2>
<p>Toolkit for processing sequences in FASTA/Q formats.</p>
<p><strong>Versions</strong>: 1.3</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>seqtk/1.3</code></p>
<h2 id="sickle">sickle<a class="headerlink" href="#sickle" title="Permanent link">&para;</a></h2>
<p>Sickle is a tool that uses sliding windows along with quality and length
thresholds to determine when quality is sufficiently low to trim the
3'-end of reads and also determines when the quality is sufficiently
high enough to trim the 5'-end of reads.</p>
<p><strong>Versions</strong>: 1.33</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>sickle/1.33</code></p>
<h2 id="slurm">slurm<a class="headerlink" href="#slurm" title="Permanent link">&para;</a></h2>
<p>Slurm is an open source, fault-tolerant, and highly scalable cluster
management and job scheduling system for large and small Linux clusters.
Slurm requires no kernel modifications for its operation and is
relatively self-contained. As a cluster workload manager, Slurm has
three key functions. First, it allocates exclusive and/or non-exclusive
access to resources (compute nodes) to users for some duration of time
so they can perform work. Second, it provides a framework for starting,
executing, and monitoring work (normally a parallel job) on the set of
allocated nodes. Finally, it arbitrates contention for resources by
managing a queue of pending work.</p>
<p><strong>Versions</strong>: 22-05-6-1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>slurm/22-05-6-1</code></p>
<h2 id="smartdenovo">smartdenovo<a class="headerlink" href="#smartdenovo" title="Permanent link">&para;</a></h2>
<p>SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT)
data.</p>
<p><strong>Versions</strong>: master</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>smartdenovo/master</code></p>
<h2 id="sortmerna">sortmerna<a class="headerlink" href="#sortmerna" title="Permanent link">&para;</a></h2>
<p>SortMeRNA is a program tool for filtering, mapping and OTU-picking NGS
reads in metatranscriptomic and metagenomic data</p>
<p><strong>Versions</strong>: 2017-07-13</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>sortmerna/2017-07-13</code></p>
<h2 id="trimmomatic">trimmomatic<a class="headerlink" href="#trimmomatic" title="Permanent link">&para;</a></h2>
<p>A flexible read trimming tool for Illumina NGS data.</p>
<p><strong>Versions</strong>: 0.39</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>trimmomatic/0.39</code></p>
<h2 id="ucx">ucx<a class="headerlink" href="#ucx" title="Permanent link">&para;</a></h2>
<p>a communication library implementing high-performance messaging for
MPI/PGAS frameworks</p>
<p><strong>Versions</strong>: 1.13.1</p>
<p><strong>Arches</strong>: generic</p>
<p><strong>Modules</strong>: <code>ucx/1.13.1</code></p>







  
    
  
  
    
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1-2.1-2M12.5 7v5.2l4 2.4-1 1L11 13V7h1.5M11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2v1.8Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">June 26, 2024</span>
  </span>

    
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Created">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M14.47 15.08 11 13V7h1.5v5.25l3.08 1.83c-.41.28-.79.62-1.11 1m-1.39 4.84c-.36.05-.71.08-1.08.08-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8c0 .37-.03.72-.08 1.08.69.1 1.33.32 1.92.64.1-.56.16-1.13.16-1.72 0-5.5-4.5-10-10-10S2 6.5 2 12s4.47 10 10 10c.59 0 1.16-.06 1.72-.16-.32-.59-.54-1.23-.64-1.92M18 15v3h-3v2h3v3h2v-3h3v-2h-3v-3h-2Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">June 26, 2024</span>
  </span>

    
    
    
  </aside>





                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
    
      
      <nav class="md-footer__inner md-grid" aria-label="Footer" >
        
          
          <a href="../" class="md-footer__link md-footer__link--prev" aria-label="Previous: Index">
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
            </div>
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Previous
              </span>
              <div class="md-ellipsis">
                Index
              </div>
            </div>
          </a>
        
        
          
          <a href="../modules/" class="md-footer__link md-footer__link--next" aria-label="Next: Module System">
            <div class="md-footer__title">
              <span class="md-footer__direction">
                Next
              </span>
              <div class="md-ellipsis">
                Module System
              </div>
            </div>
            <div class="md-footer__button md-icon">
              
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
            </div>
          </a>
        
      </nav>
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2023 The Regents of the University of California, Davis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
    <a href="mailto:hpc-help@ucdavis.edu" target="_blank" rel="noopener" title="Email HPCCF Support" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 12.13c-.15.01-.29.06-.39.17l-1 1 2.05 2 1-1c.22-.21.22-.56 0-.77l-1.24-1.23a.557.557 0 0 0-.38-.17m-2 1.75L13 19.94V22h2.06l6.06-6.07M20 4H4a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h7v-.89l8.24-8.22c.47-.49 1.12-.76 1.8-.76.34 0 .68.06 1 .19V6c0-1.12-.92-2-2.04-2m0 4-8 5-8-5V6l8 5 8-5"/></svg>
    </a>
  
    
    
    
    
    <a href="https://ucdavis.slack.com/archives/C053NKA88" target="_blank" rel="noopener" title="#hpc Channel on ucdavis.slack.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["content.code.annotate", "navigation.sections", "navigation.expand", "navigation.indexes", "navigation.top", "navigation.footer", "navigation.tracking", "search.highlight", "search.share", "search.suggest"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.ad660dcc.min.js"></script>
      
        <script src="../../javascripts/mathjax.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
    
  </body>
</html>